lib/Parser/parser.ml

```diff
@@ -0,0 +1,39 @@
+exception Wrong_number_of_args
+
+let parse_argument_line_to_array (l : string) : int array =
+  let str_list = String.split_on_char ' ' l in
+  let int_list = List.map (fun x -> int_of_string x) str_list in
```

Comment:
I come back here on how to encode variables in the text file. If we have "1112" to say color 11 row 1 column 2, how do we differentiate it from color 1 row 11 column 2 (or other variants). I really think the coordinates are too much in the model (column and row numbers are implicit, so not mandatory to specify).